app/models/service_update.rb

```diff
@@ -18,6 +18,9 @@ def self.all
   end
 
   def self.recent_updates
-    all.select { |service_update| service_update.date > 1.month.ago }
+    display_updates = []
+    recent_updates = all.select { |service_update| service_update.date > 1.month.ago }
+
+    display_updates << recent_updates[0] << recent_updates[1]
```

Comment:
What happens if `recent_updates` has less than 2 elements? I think you would get `nil` values in the result. Would `recent_updates[0, 2]` would handle this better?

Reply:
cheers Steve, have implemented as you suggested which makes more sense!